src/bolero/support.rs

```diff
@@ -348,15 +348,18 @@ impl ValueGenerator for UniqueV6InterfaceAddressGenerator {
 pub const ALPHA_NUMERIC_CHARS: &str =
     "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
 
-pub struct MacAddrString(pub String);
+pub struct UnicastMacAddrString(pub String);
 // Only generate lower case hex characters for mac addresses
 // because we cannot customize PartialEq for generated types
 // that use this, and we want to be able to compare generated
 // mac addresses with each other without concern for case.
-impl TypeGenerator for MacAddrString {
+impl TypeGenerator for UnicastMacAddrString {
     fn generate<D: Driver>(d: &mut D) -> Option<Self> {
         // Generate a random 48-bit MAC address
-        let mac = d.gen_u64(Bound::Included(&1), Bound::Excluded(&0xffff_ffff_ffff_u64))?;
+        // Set the least significant bit of the mac to 0 for unicast
+        let mac = d.gen_u64(Bound::Included(&1), Bound::Excluded(&0xffff_ffff_ffff_u64))?
+            & 0xffff_ffff_fffe;
```

Comment:
I think this is incorrect.  I think you want to unset the least significant bit of the first octet of the mac address to make it unicast.